### PR TITLE
Improve support properties computation

### PR DESCRIPTION
**Pull request by markelog**
* Remove div from the memory if it is not needed anymore

* Make `computeStyleTests` method a singleton

### CODE REVIEW

File: src/css/support.js [hunk 1]
```diff
@@ -38,6 +44,8 @@ define( [
 
 		var divStyle = window.getComputedStyle( div );
 		pixelPositionVal = divStyle.top !== "1%";
+
+		// Support: Android 4.0 - 4.3 only, Firefox <=3 - 44+
```
Comment (mgol) [inline src/css/support.js]: Did ypu check Firefox 3? :)

**Comment by mgol** [comment]
What's the size difference?

File: src/css/support.js (hunk 1 above)
Comment (markelog) [inline src/css/support.js]: Did you? Since you are the author of that comment :), i just moved it around

**Comment by markelog** [comment]
The whole byte!

**Comment by timmywil** [comment]
I like it :+1:

File: src/css/support.js (hunk 1 above)
Comment (mgol) [inline src/css/support.js]: Ah, OK then. I didn't seem the whole context on mobile so I thought this
line is new. :)

-- 
Michał Gołębiowski

**Comment by mgol** [comment]
Looks nice!

-- 
Michał Gołębiowski

**Pull request closed without merging**